imageGal1.css

```diff
@@ -1,31 +1,27 @@
 
-.mainGallery{
+#mainGallery{
     display:flex;
     justify-content:space-around;
     /* width:100%; */
 }
 
-   .subGallery{
+.sub_gallery{
```

Comment:
.sub-gallery

Sorry i may have confused you with mention of snake case. We use hyphen case for classes